packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -292,6 +294,19 @@ export const Popper: React.FunctionComponent<PopperProps> = props => {
           fn: handleUpdate,
         },
         popperInitialPositionFix.modifier,
+
+        autoSize && maxSize,
+        autoSize && {
+          name: 'applyMaxSize',
+          enabled: true,
+          phase: 'beforeWrite' as PopperJs.ModifierPhases,
+          requires: ['maxSize'],
+          fn({ state }) {
+            const { height, width } = state.modifiersData.maxSize;
+            state.styles.popper.maxHeight = `${height}px`;
+            state.styles.popper.maxWidth = `${width}px`;
```

Comment:
Q: _How this will be cleaned up?_
A: It works, styles are removed.

A snippet for testing:

```tsx
import * as React from "react";
import {
  Button,
  Checkbox,
  Popup,
  Image,
  Flex,
  Text
} from "@fluentui/react-northstar";
import { MoreIcon } from "@fluentui/react-icons-northstar";

const PopupExampleAutoSize = () => {
  const [autoSize, setAutoSize] = React.useState();
  const [open, setOpen] = React.useState();

  return (
    <div
      style={{
        paddingTop: "10rem",
        maxWidth: "40rem",
        border: "red dashed",
        position: "relative",
        overflow: "hidden"
      }}
    >
      <Checkbox
        label="Autosize"
        checked={autoSize}
        onChange={(e, data) => setAutoSize(data.checked)}
      />
      <Checkbox
        label="Open"
        checked={open}
        onChange={(e, data) => setOpen(data.checked)}
      />
      <Popup
        trigger={<Button icon={<MoreIcon />} title="Show popup" />}
        content={
          <Flex column gap="gap.small">
            <Image src="https://fabricweb.azureedge.net/fabric-website/assets/images/wireframe/square-image.png" />
            <Text content="Lorem ipsum dolor" />
          </Flex>
        }
        open={open}
        position={"above"}
        inline
        autoSize={autoSize}
      />
    </div>
  );
};

export default PopupExampleAutoSize;
```